crates/bevy_render/macros/src/as_bind_group.rs

```diff
@@ -161,6 +161,29 @@ pub fn derive_as_bind_group(ast: syn::DeriveInput) -> Result<TokenStream> {
                             }
                             _ => {}
                         }
+
+                        let binding_array_binding = binding_array_binding.unwrap_or(0);
```

Comment:
0? Why is 0 implicitly correct? I thought the data is automatically serialised into a buffer at binding 0?

Reply:
It's not. I just need a dummy value to insert into `bindless_binding_layouts`, because we populate that array anyway even in the non-bindless case, for simplicity. Maybe that should be refactored though.